### DISCUSSION TITLE
Create token list Modal with wagmi?

### DISCUSSION THREAD

**Discussion by dev-cz** [wevm/wagmi#681] [General]
Was anyone successfully integrated wagmi with a token list modal, like one that Uniswap has when selecting tokens to swap between?

Looking for the best way to incorporate this into my dApp I am building but want to see what others have done before going down a black hole. :)

**Comment by matthewlilley** [comment]
@dev-cz we have a redux-token-list package for now, but made the same suggestion that it would be great if this was natively supported by wagmi in future.

In-case you're interested:

https://github.com/sushiswap/sushiswap/tree/master/packages/redux/token-lists
